### PR TITLE
Update preset-security-policies.md

### PR DESCRIPTION
**Pull request by andypunt**
Added clarification to customer versus default policy, which did not change.

### CODE REVIEW

**Comment by yogkumgit** [comment]
@JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks

File: microsoft-365/security/office-365-security/preset-security-policies.md [hunk 1]
```diff
@@ -83,7 +83,8 @@ When multiple policies are applied to a user, the following order is applied fro
 
 1. **Strict protection** preset security policy
 2. **Standard protection** preset security policy
-3. Any other related policies.
+3. Custom policies
+4. Default Policy
```
Comment (JohanFreelancer9) [inline microsoft-365/security/office-365-security/preset-security-policies.md]:
```suggestion
4. Default policy
```

File: microsoft-365/security/office-365-security/preset-security-policies.md [hunk 2]
```diff
@@ -83,7 +83,8 @@ When multiple policies are applied to a user, the following order is applied fro
...
 
 In other words, the settings of the **Strict protection** policy override the settings of the **Standard protection** policy, which overrides the settings from any other related policies.
```
Comment (JohanFreelancer9) [inline microsoft-365/security/office-365-security/preset-security-policies.md]:
```suggestion
In other words, the settings of the **Strict protection** policy override the settings of the **Standard protection** policy, which overrides the settings from custom policies and the default policy.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Pull request merged**